components/compliance-service/compliance.go

```diff
@@ -177,6 +178,10 @@ func serveGrpc(ctx context.Context, db *pgdb.DB, connFactory *secureconn.Factory
 		if err != nil {
 			logrus.WithError(err).Fatal("could not register project update task executors")
 		}
+		err = project_purge.RegisterTaskExecutors(cerealProjectUpdateManager, "compliance", ingesticESClient)
```

Comment:
Extreme nit: we name this var `projectUpdateManager` elsewhere and I like the idea of it being the same 🤷‍♂ 🙃